portalnet/Cargo.toml

```diff
@@ -50,6 +50,7 @@ validator = { version = "0.13.0", features = ["derive"] }
 url = "2.3.1"
 utp-rs = { git = "https://github.com/ethereum/utp", tag = "v0.1.0-alpha.10" }
 tokio-stream = { version = "0.1.14", features = ["sync"] }
+itertools = "0.12.1"
```

Comment:
can you move this so the list is sorted alphabetically

Reply:
I used `$cargo add ...` which keeps dependencies sorted, so I wasn't paying attention.
The thing is that it keeps dependencies sorted only if they are already sorted.
Fixed everything now.